lib/ntbb-database.lib.php

```diff
@@ -27,12 +27,20 @@ function connect() {
 				$this->db = new PDO(
 					"mysql:dbname={$this->database};host={$this->server};charset={$this->charset}",
```

Comment:
Perhaps we should add a `wait_timeout=SECONDS` value to ensure these connections close after a period of inactivity? This isn't exactly my area of expertise so correct me if I'm wrong.